.docker/opera/supervisord.conf

```diff
@@ -0,0 +1,22 @@
+[program:opera]
+environment=HOME="/home/%(ENV_USER)s",USER="%(ENV_USER)s",DISPLAY="%(ENV_DISPLAY)s"
+command=/usr/bin/opera --no-sandbox --window-position=0,0 --display=%(ENV_DISPLAY)s --user-data-dir=/home/neko/.config/opera --no-first-run --start-maximized --bwsi --force-dark-mode --disable-file-system --disable-gpu --disable-software-rasterizer --disable-dev-shm-usage
```

Comment:
If opera does not accept chromium policies, maybe most of the args are as well unused.

Reply:
I've removed the args that don't result in Opera crashing or errors being output to the logs.